### ISSUE TITLE
revoking the token should restart the authorization process

### ISSUE THREAD

**Issue by ccoenen** [muesli/mastotool#10]
After successfully running this script (first-run experience is quite nice!), I revoked my access token.

This is handled relatively gracefully, the terminal just outputs

> mastotool.exe stats
> Can't retrieve user: bad request: 401 Unauthorized: The access token was revoked

I think at this point a new authorisation should be _offered_. (not forced, the failure could have different reasons as well).

**Comment by muesli** [comment]
You're right, we can do better and handle this situation appropriately. In the meantime, you can just delete the entire config file `mastotool.json`, which would obviously restart the auth process.
